CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# CONTRIBUTING
+
+Contributions are always welcome, no matter how large or small. Before contributing,
+please read the [code of conduct](CODE_OF_CONDUCT.md).
+
+## Setup
+
+1. **Download this repository and go to its folder**
+
+    ```shell
+     git clone git@github.com:Qiskit/qiskit.org.git && cd qiskit.org
+    ```
+
+2. **Install dependencies**
+
+    ```shell
+   npm install
+    ```
+
+3. **Run the development server with hot reload at [localhost:3000](localhost:3000)**
+
+   ```shell
+   npm run dev
+   ```
+
+## Available scripts
+
+Run a local server enabling inspector agent:
+```shell
+  npm run dev-debug
+```
+
+Run unit tests made with [Jest](https://jestjs.io/):
+```shell
+  npm run test
+```
+
+Build static version ready for production, output will generated inside a new folder called `dist`:
```

Comment:
Our `dist` directory is called `public` instead.

Reply:
Fixed in the last commit 😊 thanks!